### PR TITLE
Changed warning trace to error if color scheme by layer is choosen for MVD2 circuits

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: brayns/io/CircuitLoader.cpp [hunk 1]
```diff
@@ -307,10 +307,10 @@ class CircuitLoader::Impl
         }
         catch (...)
         {
-            BRAYNS_WARN << "Only MVD2 format is currently supported by Brion "
```
Comment (tribal-tec) [inline brayns/io/CircuitLoader.cpp]: Just print the message if ' Color scheme by layer' was requested? Then it should be an error.

Comment (favreau) [reply brayns/io/CircuitLoader.cpp]: Fair enough.

Comment (tribal-tec) [reply brayns/io/CircuitLoader.cpp]: Now the commit message and the PR message don't match anymore.

Comment (karjonas) [reply brayns/io/CircuitLoader.cpp]: Why do you do this check in the try/catch and not before? This hides other exceptions then.

**Pull request merged**